### DISCUSSION TITLE
Add option to choose between real-time UTC and in-sim UTC

### DISCUSSION THREAD

**Discussion by Phil787-9** [magknight/787aviatorsTracker#840] [Ideas]
**Is your feature request related to a problem? Please describe.**
In older versions of the MK, the clock would display in-sim UTC, regardless of the actual real-time UTC. In the newer versions (1.8.3+), only real-life UTC is displayed regardless of sim time. This is quite annyoing if you fly offline and not real-time. I can always display real-time UTC on external clocks (e.g. my phone, watch, or computer) if needed, but I cannot do that with in-sim UTC. So, please bring back in-sim UTC. Thanks!

**Describe the solution you'd like**
Include an option in the EFB that would allow the user to choose between the display of real-time UTC and in-sim UTC. 

**Describe alternatives you've considered**
Right now, I have to open the sim menu and consequently pause the flight in order to view the in-sim UTC. This is rather annyoing and time-consuming, as I frequently lost my time-awareness.

**Additional context**
N/A

**Comment by aeroniemi** [comment]
this won't be added as a feature, as quite a lot of things rely on linear time, which x-plane's sim time isn't (you can go back in time, or accelerate it)

**Comment by Phil787-9** [comment]
It was possible just a few versions ago. What changed, if I may ask?

**Comment by aeroniemi** [comment]
we started caring about time - flight state detection, such as is used by the cdu and ND maps required that (and has done since well before 1.7.0). we simply updated all of the time displays to show the same time, rather than some showing utc and others sim time.

**Comment by Phil787-9** [comment]
So, it's possible, you just don't want to do it?

**Comment by aeroniemi** [comment]
I mean if you want all of the clocks to not display the same time then I guess, but that'll make even less sense if the cdu times get updated

**Comment by Phil787-9** [comment]
I'd appreciate at least having to option to decide for myself which time I'd like to see displayed (if it's not too much of an imposition)
